pydev_ipython/inputhookqt4.py

```diff
@@ -19,7 +19,8 @@
 import os
 import signal
 
-from _pydev_bundle._pydev_saved_modules import threading
+import threading
```

Comment:
Why was this change needed? Usually it's preferred to use the pydevd import as it'll get the original module when using gevent.

Reply:
I didn't make this change, it was in debugpy. I'll revert it.